### ISSUE TITLE
Question: How to update plug-ins

### ISSUE THREAD

**Issue by figihiuy** [Acly/krita-ai-diffusion#127]
How do I update the plugin? Is it basically just overwrite?
Or do I have to re-install it?

**Comment by Acly** [comment]
Yep, just overwrite. Maybe one day there will be some kind of auto-updater.

**Comment by Duemellon** [comment]
For me when I copied over the files I had to have Krita completely reset before it would work with the plugin again. I'm on Win11. Just an FYI.
